examples/jsm/loaders/GLTFLoader.js

```diff
@@ -4559,6 +4561,12 @@ function addPrimitiveAttributes( geometry, primitiveDef, parser ) {
 
 	}
 
+	if ( ColorManagement.workingColorSpace !== LinearSRGBColorSpace && 'COLOR_0' in attributes ) {
+
+		console.warn( `THREE.GLTFLoader: Converting vertex colors from "srgb-linear" to "${ColorManagement.workingColorSpace}" not supported.` );
+
+	}
```

Comment:
It looks like converting color attributes was lost in the new roadmap in https://github.com/mrdoob/three.js/issues/26479. We should plan to cover this case in the long term, right?

Reply:
Yes, I do intend that to be part of step 2.4, just not ready to make those changes here yet.

Reply:
Or did you mean just handling sRGB vs Linear sRGB vertex colors in existing workflows, without this wide gamut work? Are we missing something there, if so?

Reply:
I see - I think I expect there to be more to this color vertex attribute issue than just updating the loaders. I think item 3.3 we added in #23614 captured what I imagine being needed more completely. For example if users are dynamically generating vertex colors for geometry they need a way to ensure it's converted to the working color space. Though I guess that's convenience more than anything at this point since users can use the `Color` instance convert before constructing the vertex data array 🤔

Either way I think this is a good PR!

Reply:
Ah yeah. This is kind of verbose and unintuitive today:

```javascript
const _color = new THREE.Color();
const color = geometry.attributes.color;
for (let i = 0; i < color.count; i++) {
  _color.setRGB(color.getX(i), color.getY(i), color.getZ(i), inputColorSpace);
  color.setXYZ(i, ..._color);
}
```

I don't know if I like the idea of a color-specific BufferAttribute subclass, but a cleaner way to do this would be nice.